nilearn/glm/contrasts.py

```diff
@@ -262,6 +265,35 @@ def p_value(self, baseline=0.0):
         self.p_value_ = p_values
         return p_values
 
+    def one_minus_pvalue(self, baseline=0.0):
+        """Return a parametric estimate using
+        the cumulative distribution function of the p-value associated
+        with the null hypothesis: (H0) 'contrast equals baseline'
+
+        Parameters
+        ----------
+        baseline : float, optional
+            baseline value for the test statistic
+
+        Returns
+        -------
+        one_minus_pvalues : 1-d array, shape=(n_voxels,)
+            one_minus_pvalues, one per voxel
+        """
+        if self.stat_ is None or not self.baseline == baseline:
+            self.stat_ = self.stat(baseline)
+        # Valid conjunction as in Nichols et al, Neuroimage 25, 2005.
+        if self.contrast_type == 't':
+            one_minus_pvalues = sps.t.cdf(self.stat_,
+                                          np.minimum(self.dof, self.dofmax))
+        elif self.contrast_type == 'F':
+            one_minus_pvalues = sps.f.cdf(self.stat_, self.dim,
+                                          np.minimum(self.dof, self.dofmax))
+        else:
+            raise ValueError('Unknown statistic type')
```

Comment:
could you add a test for this case?

Reply:
This is basically a replication of the previous function `p_value` but using cdf instead of sf. I don't find a concrete test function for `p_value` that I could also replicate for this one. What do you suggest?

Reply:
I understand that this line has not been tested and that's why codecov/patch is complaining. I think I should add a test where also `p_value` function is tested but I don't find it.

Reply:
I would suggest passing an invalid value as `contrast_type` to the model's `__init__`, then calling `one_minus_pvalue`

Reply:
It seems that the second test of the function `test_one_minus_pvalue` is not working, since line 293 was not evaluated not evaluated. Any suggestion?

Reply:
line 293 will never run because an exception would be raised in `stat` before it is reached -- it's ok if it is not covered by the tests.

Reply:
if you want to avoid it you can do for example
```
if contrast type == t:
  ...
else:
  assert contrast_type == f
  ...
```
but it's ok the way it is, and it is what is done in p_value so you can just keep it

Reply:
I did what you suggested in my last commit and the problem is solved! But there's still a red segment in Codecov Report...